eng/pipelines/VS-Insertion-release.yml

```diff
@@ -0,0 +1,81 @@
+# Inserts the binaries generated by VS-release.yml into VS repo
+---
+name: $(Date:yyyMMdd).$(Rev:r)
+
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+  pipelines:
+  - pipeline: MIEngine_MDD
+    source: 'MIEngine_VS_Release'
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool:
+      name: VSEngSS-MicroBuild2022-1ES
+      os: windows
+    sdl:
+      sourceAnalysisPool:
+        name: VSEngSS-MicroBuild2022-1ES
+        os: windows 
+    stages:
+    - stage: VS_Insertion
+      displayName: VS Insertion
+      jobs:
+      - job:
+        templateContext:
+          inputs:
+          - input: pipelineArtifact
+            artifactName: PackageVersion
+            targetPath: $(Build.ArtifactStagingDirectory)\PackageVersion
+            pipeline: MIEngine_MDD
+        steps:
+        - checkout: none
+
+        - powershell: |
+            $version= [IO.File]::ReadAllText("$(Build.ArtifactStagingDirectory)\PackageVersion\NugetPackageVersion.txt")
+            Write-Host "##vso[task.setvariable variable=MDDPackageVersion;]$version"
+          displayName: 'Set MDDPackage Version'
+
+        - task: ms-vseng.MicroBuildShipTasks.55100717-a81d-45ea-a363-b8fe3ec375ad.MicroBuildInsertVsPayload@4
+          displayName: 'Insert VS Payload'
+          inputs:
+            TargetBranch: $(TargetBranch)  
+            TeamName: 'VS Debugger Platform'
+            TeamEmail: $(TEAMEMAIL)
+            DefaultConfigValues: 'VS.Redist.Debugger.MDD.MIEngine=$(MDDPackageVersion)'
```

Comment:
>DefaultConfigValues: 'VS.Redist.Debugger.MDD.MIEngine=$(MDDPackageVersion)' [](http://example.com/codeflow?start=12&length=75)

Is this still correct? Or there a different way to update src/ConfigData?

Reply:
Yep. This modifies `src/ConfigData/Packages/misc.props`